src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,101 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
```

Comment:
Please rework your solution without using existing Java collections in order to get more understanding of methods principles.
ArrayList -> array (we don't need any resize features of ArrayList here)
LinkedList -> raw nodes usage (we don't need "previous" property here, next is enough)